libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/component/recenter/RecenterButtonComponentContractImplTest.kt

```diff
@@ -0,0 +1,79 @@
+package com.mapbox.navigation.dropin.component.recenter
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.dropin.util.TestStore
+import com.mapbox.navigation.testing.MainCoroutineRule
+import com.mapbox.navigation.ui.app.internal.camera.CameraAction
+import com.mapbox.navigation.ui.app.internal.camera.TargetCameraMode
+import com.mapbox.navigation.ui.app.internal.navigation.NavigationState
+import io.mockk.mockk
+import io.mockk.spyk
+import io.mockk.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.yield
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@OptIn(ExperimentalPreviewMapboxNavigationAPI::class, ExperimentalCoroutinesApi::class)
+class RecenterButtonComponentContractImplTest {
+
+    @get:Rule
+    var coroutineRule = MainCoroutineRule()
```

Comment:
Why is it var?

Reply:
Probably because I copy pasted it from another test class that already had it set as `var`. 
TBH... only the JUnit test runner interacts with this class and it doesn't really matter what visibility those fields are.